libs/webb-ui-components/src/components/TransactionInputCard/TransactionInputCard.tsx

```diff
@@ -0,0 +1,337 @@
+'use client';
```

Comment:
Do we need to use `use client` in`webb-ui-components`?

Reply:
Indeed, since some components will be utilized in the Next app, it is beneficial to identify whether a new component is for the server or the client. Additionally, we need to address the [task of reviewing the web-ui-kit](https://github.com/webb-tools/webb-dapp/issues/1491) and labeling which components are server components and which are client components.